samples/ElmInspiredOne/ElmInspiredOneXF.iOS/AppDelegate.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UIKit;
+using Foundation;
+using Gjallarhorn.Bindable.Framework;
+using Program = ElmInspiredOne.Program;
+using ElmInspiredXF;
+
+namespace ElmInspiredOneXF.iOS
+{
+    // The UIApplicationDelegate for the application. This class is responsible for launching the
+    // User Interface of the application, as well as listening (and optionally responding) to application events from iOS.
+    [Register("AppDelegate")]
+    public class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
+    {
+        // class-level declarations
+
+        public override UIWindow Window
+        {
+            get;
+            set;
+        }
+
+        public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
+        {
+            global::Xamarin.Forms.Forms.Init();
+            //var info = Framework.CreateApplicationInfo(Program.applicationCore, new MainPage());
```

Comment:
These lines were copied from the Android app shell project.
Unsure the correct interface to load into an iOS application.
is there an interface that can bind to UIApplication somehow?